src/scale.ts

```diff
@@ -382,9 +382,13 @@ export interface Scale {
    */
   domain?: Domain;
 
+
+  // Hide because we might not really need this.
```

Comment:
Not just that.  Thinking more I think I prefer a) but don't want to make a call for it right now. 
By hiding it, we can make this decision in the future if we need to.